wandb/sdk/internal/internal_api.py

```diff
@@ -1346,7 +1360,12 @@ def no_retry_4xx(e):
             if entity:
                 self.set_setting("entity", entity["name"])
 
-        return response["upsertSweep"]["sweep"]["name"]
+        if mutation_version_used >= 3:
```

Comment:
keeping track of mutation used is a bit awkward, how about just:
```
warnings = response["upsertSweep"].get("configValidationWarnings", [])
```

Reply:
done